services/prompter/user-prompts.js

```diff
@@ -0,0 +1,72 @@
+const PromptUtils = require('./prompt-utils');
+
+class UserPrompts extends PromptUtils {
+  constructor(envConfig, prompts, requests) {
+    super(requests);
+    this.envConfig = envConfig;
+    this.prompts = prompts;
+    this.FORMAT_PASSWORD = /^(?=\S*?[A-Z])(?=\S*?[a-z])((?=\S*?[0-9]))\S{8,}$/;
+  }
+
+  handlePrompts() {
+    this.handleEmail();
+    this.handleCreatePassword();
+    this.handlePassword();
```

Comment:
Not handled in the first implementation. It is set here : `token: program.token`, from general prompter (taken from first implementation)

Reply:
<img width="542" alt="Screenshot 2019-11-13 at 10 42 37" src="https://user-images.githubusercontent.com/12433882/68751704-542a9700-0602-11ea-9d03-3021f200c942.png">

This can be merged and handle directly in a token handler in user prompt.

Reply:
After checking, doing the refacto can have side effects. This PR is already HUGE. Let's do this in another card.